l2geth/scripts/start.sh

```diff
@@ -4,18 +4,20 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
 REPO=$DIR/..
 
 IS_VERIFIER=
-ROLLUP_SYNC_SERVICE_ENABLE=true
+ROLLUP_SYNC_SERVICE_ENABLE=false
```

Comment:
Is this done for debugging?

Reply:
This turned off polling for deposits so that the system can be ran without having a reliance on an L1 node